pandas/plotting/_matplotlib/core.py

```diff
@@ -1,5 +1,5 @@
 import re
-from typing import Optional  # noqa
+from typing import Optional  # noqa:F401
```

Comment:
This shouldn't need the noqa; might be a historical artifact when flake8 didn't parse type comments correctly